README.md

```diff
@@ -1,22 +1,21 @@
-# DSC Devsprints Jan 2020
```

Comment:
Don't remove the markdown tags. The hashes are for markdown. Just revert this commit by doing `git revert <this_commit_hash>`